samples/dnn/ssd_object_detection.cpp

```diff
@@ -126,9 +96,8 @@ int main(int argc, char** argv)
             cvtColor(frame, frame, COLOR_BGRA2BGR);
 
         //! [Prepare blob]
-        Mat preprocessedFrame = preprocess(frame);
-
-        Mat inputBlob = blobFromImage(preprocessedFrame, 1.0f, Size(), Scalar(), false); //Convert Mat to batch of images
+        //preprocess and convert frame to a series of images
+        Mat inputBlob = blobFromImage(frame, 1.0f, Size(), Scalar(), false, true);  //returns 4-dimensional blob
```

Comment:
At the https://github.com/opencv/opencv/issues/9967#issuecomment-343419551 I meant to use `blobFromImage` to do equal preprocessing steps (resize, subtract mean).